index.html

```diff
@@ -1011,6 +1011,47 @@ <h4>Exchange Examples</h4>
     <h2>Privacy Considerations</h2>
     <p>
     </p>
+
+    <section>
+      <h3>Delegation</h3>
+
+      <p>
+<a>Verifiable credentials</a> [[VC-DATA-MODEL-2.0]] are a standard data model
+designed to mitigate risks of misuse and fraud. As a data model, <a>verifiable
+credentials</a> are protocol-neutral and consider at least two types of
+entities: <a>issuer</a> and <a>subject</a>. When the subject of a <a>verifiable
+credential</a> is a natural person or linked to a natural person, privacy and
+human rights can be impacted by the vastly more efficient processing of
+standardized <a>verifiable credentials</a> as compared to their analog
+ancestors.
+      </p>
+
+      <p>
+Technology, in the form of standardized APIs and protocols for issuing
+<a>verifiable credentials</a>, further enhances the efficiency of processing
+<a>verifiable credentials</a> and adds to the risks of unforeseen privacy and
+human rights consequences.
+      </p>
+
+      <p>
+<a>Verifiable credentials</a> issuance has a request phase and a delivery phase.
+The request might be made by the <a>subject</a> or another role, and delivery
+can be to a client that might or might not be controlled by the subject.
+Delegation is highly relevant for both phases. The <a>issuer</a> might delegate
+processing of the request to a separate entity. The subject, for their part,
+might also delegate the ability to request a <a>verifiable credential</a> to a
```

Comment:
Note that the subject may not always have the capability or ability to perform delegation. Examples include: a new born baby, a pet, and a person with dementia. So the request might be performed by a third party who was not delegated by the subject.

Reply:
Yes, excellent point, I'll work that into the text in the next revision.

Reply:
Done in 098da503d35150d8448a05939b301e33202af904.